ember-highcharts/src/components/high-charts.ts

```diff
@@ -203,80 +203,59 @@ export default class HighCharts<
    */
   async _importHighchartsDeps() {
     if (this.args.mode === 'Map') {
-      Highcharts = await waitForPromise(import('highcharts/highmaps'));
+      //@ts-expect-error No idea, sielnt TS
```

Comment:
I'm sorry, but no idea what it wants and don't see a cheap way to find out. Not ideal, but solves the problem.